dcs/liveries/livery.py

```diff
@@ -86,13 +86,16 @@ def from_lua(code: str, path: str) -> Optional[Livery]:
             logging.exception("Could not parse livery definition at %s", path)
             return None
         livery_name = data.get("name", path_id)
+
         countries_table = data.get("countries")
-        if countries_table is None:
-            countries = None
-        else:
-            countries = set(countries_table.values())
-        order = data.get("order", 0)
+        countries = None
+        if countries_table is not None:
+            try:
+                countries = set(countries_table.values())
+            except AttributeError:
```

Comment:
I do not have the answer to this. The users of Liberation who are encountering an issue in this part of the pydcs codebase do not report which livery causes their error, so it has been untestable. It could easily be an alternate format like a list as you suggest, or it could be something completely malformed. Ultimately the goal here is to print out the path to the failing livery and move on, as currently the users of Liberation have been reporting their attempts of diagnosis by searching through their liveries folder and removing ones until "it works".

Reply:
Ack, I just wish we could diagnose the problem so we could fix it rather than hide it. This is a step towards that, but if you had a stack trace maybe it would have had enough info to be a hint (or maybe not).